practice-two/src/components/sections/places.html

```diff
@@ -0,0 +1,116 @@
+<section class="places-section">
+  <div class="container places-container">
+    <h2 class="places-heading">Places</h2>
+
+    <div class="select-wrapper">
+      <select title="location" name="location" class="select-places-locations">
+        <option selected value="san-francisco">San Francisco</option>
+        <option value="south-liliana">South Liliana</option>
+        <option value="blickview">Blickview</option>
+        <option value="new raven">New Raven</option>
+      </select>
+    </div>
+
+    <ul class="list-places-locations">
+      <li class="places-location">
+        <input id="san-francisco" type="radio" name="location" class="location-radio">
+        <label for="san-francisco" class="location">
+          San Francisco
+        </label>
+      </li>
+      <li class="places-location">
+        <input id="south-liliana" type="radio" name="location" class="location-radio">
+        <label for="south-liliana" class="location">
```

Comment:
Don't use <input> and <label> here. Should use <a> tag. You can refer this article https://getbootstrap.com/docs/5.0/components/navs-tabs/#tabs